### PR TITLE
Ant task

### PR DESCRIPTION
**Pull request by KrasimirRaikov**
A Person class, a Demo for it, and a PersonTest, using ant to build the project

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0"?>
+<project name="Hello World Project" default="compile">
+    <property name="src.dir" value="src"/>
+    <property name="test.dir" value="test"/>
+    <property name="lib.dir" value="lib"/>
+    <property name="build.dir" value="build"/>
```
Comment (panayotkulchev) [inline build.xml]: is it possible this properties to be moved in separate file?

File: src/com/clouway/build/tools/Person.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package com.clouway.build.tools;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * @author Krasimir Raikov(raikov.krasimir@gmail.com)
+ *         Person just because
```
Comment (panayotkulchev) [inline src/com/clouway/build/tools/Person.java]: What does this line means?

File: src/com/clouway/build/tools/Demo.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com.clouway.build.tools;
+
+
```
Comment (panayotkulchev) [inline src/com/clouway/build/tools/Demo.java]: many empty lines

File: src/com/clouway/build/tools/Demo.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+/**
+ * @author Krasimir Raikov(raikov.krasimir@gmail.com)
+ *         Demo for the Person class
+ */
+public class Demo {
+
+
```
Comment (panayotkulchev) [inline src/com/clouway/build/tools/Demo.java]: empty line

File: src/com/clouway/build/tools/Person.java (hunk 2 above)
Comment (KrasimirRaikov) [inline src/com/clouway/build/tools/Person.java]: I was just trying generating javadoc, and I forgot to remove it.

File: build.xml (hunk 1 above)
Comment (KrasimirRaikov) [inline build.xml]: done.

File: src/com/clouway/build/tools/Person.java (hunk 2 above)
Comment (KrasimirRaikov) [inline src/com/clouway/build/tools/Person.java]: fixed.

File: src/com/clouway/build/tools/Demo.java (hunk 3 above)
Comment (KrasimirRaikov) [inline src/com/clouway/build/tools/Demo.java]: fixed.

File: src/com/clouway/build/tools/Demo.java (hunk 4 above)
Comment (KrasimirRaikov) [inline src/com/clouway/build/tools/Demo.java]: fixed.

**Comment by panayotkulchev** [comment]
LGTM!

**Comment by panayotkulchev** [comment]
Always wait for LGTM before push to master like you did with the previous task.

**Pull request closed without merging**